.github/workflows/release.yml

```diff
@@ -158,13 +156,13 @@ jobs:
           RUST_TARGET: ${{ matrix.target }}
       - name: Build CLI
         run: |
-          cargo build --release --features cli --target ${{ matrix.target }}
+          yarn napi build --bin lightningcss --release --features cli --target ${{ matrix.target }}
```

Comment:
Probably shouldn't be `napi` here? This is building the CLI, which is a standalone binary.

Reply:
`napi` is able to build binary, and I use napi to cross link `armv7-unknown-linux-gnueabihf` target here